src/components/events/event/EventPage.jsx

```diff
@@ -166,282 +167,566 @@ const EventPage = () => {
+                    backgroundColor: 'white',
+                    display: 'flex',
+                    flexDirection: 'column',
+                    justifyContent: 'flex-start',
+                    marginTop: '2.5em',
                   }}
                 >
-                  {parseDate().startTime} - {parseDate().endTime}
-                </p>
+                  <p className="header" style={{ paddingLeft: '1em' }}>
+                    Requirements
+                  </p>
+                  <Divider style={{ padding: 0, margin: 0, marginBottom: '1em' }} />
+                  <div
+                    style={{
+                      display: 'flex',
+                      flexDirection: 'row',
+                      flexWrap: 'wrap',
+                      paddingLeft: '1em',
+                    }}
+                  >
+                    {/* TODO: fix this */}
+                    {[eventData.requirements].map(e => {
+                      return (
+                        <div
+                          key={e}
+                          className="requirementsTag"
+                          style={{
+                            paddingLeft: '1em',
+                            paddingRight: '1em',
+                            margin: '1em',
+                            justifyContent: 'center',
+                            alignItems: 'center',
+                          }}
+                        >
+                          <p
+                            style={{
+                              padding: 0,
+                              margin: 0,
+                            }}
+                          >
+                            {e}
+                          </p>
+                        </div>
+                      );
+                    })}
+                  </div>
+                </div>
               </div>
             </div>
-            {getPostEvent()}
-            {getNotes()}
-            <div
-              style={{
-                display: 'flex',
-                flexDirection: 'column',
-                justifyContent: 'space-between',
-                backgroundColor: 'white',
-                marginTop: '4em',
-                height: '6em',
-              }}
-            >
-              <p className="header">Waivers</p>
-              <Button
-                style={{
-                  width: '13em',
-                  display: 'flex',
-                  flexDirection: 'row',
-                  alignItems: 'center',
-                  justifyContent: 'center',
-                }}
-              >
-                <VerticalAlignBottomOutlined />
-                <p style={{ padding: 0, margin: 0, paddingLeft: '.7em' }}>Click to Download</p>
-              </Button>
-              <p style={{ fontFamily: 'AvenirNextLTProBold', fontSize: '14px', color: '#888888' }}>
-                Not yet implemented
-              </p>
+          </>
+        )}
+
+        {!isAdmin && (
```

Comment:
if event is upcoming: should have a "sign up" button if the user has not signed up for the event, or an "edit event application" button if the user has already signed up for this specific event.

if event is in the past: should have a "submit hours" button if user attended the event.